crates/tx-sitter-client/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "tx-sitter-client"
+version = "0.1.0"
+edition = "2021"
+publish = false
+
+[dependencies]
+anyhow = "1.0"
+# chrono = { version = "0.4.23", features = ["serde"] }
+# cognitoauth = { path = "../cognitoauth" }
+ethers = { version = "2.0.10", features = [ ] }
+# hyper = { version = "^0.14.17", features = ["server", "tcp", "http1", "http2"] }
+reqwest = "0.11.14"
+serde = { version = "1.0.154", features = ["derive"] }
+serde_json = "1.0.94"
+# thiserror = "1.0.39"
+# tokio = { version = "1.17", features = ["full"] }
+# tracing = "0.1.37"
+# url = "2.3.1"
```

Comment:
Are these commented out deps going to be needed?

Reply:
Good point, thanks!